.travis.yml

```diff
@@ -11,3 +11,4 @@ sudo: false
 notifications:
   email:
     - iluwatar@gmail.com
+    - hemant.bothra15@gmail.com
```

Comment:
If we introduce Github Actions CI then we should get rid of Travis build. There's this coveralls test coverage report that should be migrated too. At the moment Github Actions seems to do `mvn clean compile` vs `mvn clean test jacoco:report coveralls:report` in Travis.

Reply:
Yup, for the coverage part I was thinking to go with Sonar which we are having #26 
Was thinking to get rid of travis post we fix the sonar part

Reply:
Yes, that's ok then